modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/StoriesPage.kt

```diff
@@ -264,13 +265,15 @@ private fun ShareButton(
     modifier: Modifier = Modifier,
 ) {
     RowOutlinedButton(
-        text = stringResource(id = LR.string.share),
-        border = BorderStroke(ShareButtonStrokeWidth, Color.White),
+        text = stringResource(id = LR.string.end_of_year_share_story),
+        border = null,
         colors = ButtonDefaults
             .outlinedButtonColors(
                 backgroundColor = Color.Transparent,
                 contentColor = Color.White,
             ),
+        fontSize = 14.sp,
```

Comment:
Do we maybe want to disable scaling on this button like we do on the rest of the story? I noticed that on the largest font sizes the button jumped out because it was much larger than the rest of the text on the screen. Or maybe we want to respect the user's font scale on the button. I don't know, but I wanted to mention it.

Reply:
Let me try it out in my next PR.